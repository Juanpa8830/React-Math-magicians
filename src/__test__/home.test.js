import React from 'react';
import renderer from 'react-test-renderer';
import HomeSection from '../pages/home';

test('fit home snapshot', () => {
  const tree = renderer.create(<HomeSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
