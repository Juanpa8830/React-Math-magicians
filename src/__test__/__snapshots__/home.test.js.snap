// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fit home snapshot 1`] = `
<div
  className="home-section"
>
  <h2
    className="home-title"
  >
    Welcome to our page!
  </h2>
  <p
    className="textSection"
  >
    <p>
      Lorem ipsum dolor sit amet consectetur adipiscing elit. Quisque faucibus ex sapien vitae pellentesque sem placerat. In id cursus mi pretium tellus duis convallis. Tempus leo eu aenean sed diam urna tempor. Pulvinar vivamus fringilla lacus nec metus bibendum egestas. Iaculis massa nisl malesuada lacinia integer nunc posuere. Ut hendrerit semper vel class aptent taciti sociosqu. Ad litora torquent per conubia nostra inceptos himenaeos.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipiscing elit. Quisque faucibus ex sapien vitae pellentesque sem placerat. In id cursus mi pretium tellus duis convallis. Tempus leo eu aenean sed diam urna tempor. Pulvinar vivamus fringilla lacus nec metus bibendum egestas. Iaculis massa nisl malesuada lacinia integer nunc posuere. Ut hendrerit semper vel class aptent taciti sociosqu. Ad litora torquent per conubia nostra inceptos himenaeos.
    </p>
  </p>
</div>
`;
